cmd/zhack/zhack.c

```diff
@@ -133,8 +133,10 @@ import_pool(const char *target, boolean_t readonly)
 	char *name;
 
 	kernel_init(readonly ? FREAD : (FREAD | FWRITE));
-	g_zfs = libzfs_init();
-	ASSERT(g_zfs != NULL);
+	if ((g_zfs = libzfs_init()) == NULL) {
+		(void) fprintf(stderr, "%s", libzfs_error_init(errno));
+		return;
```

Comment:
Yes, yes it should be.  I'll fix it by reverting this hunk so it matches upstream.  I clearly just replaced all this calls without fully inspecting them.